tests/phpunit/data/fiximageswithexternalurl-output.wikitext

```diff
@@ -0,0 +1,3 @@
+[[File:Some test file]]
+
+<img src="https://server/confluence/download/attachments/123456789/image2021-8-17_10-15-14.png?version=1&modificationDate=1914846492120&api=v2" height="150" />
```

Comment:
The goal was to convert this to `[[File:<whatever-target-filename-is-used-for-123456789/image2021-8-17_10-15-14.png|x150]]`

Of course only if the attachment `123456789/image2021-8-17_10-15-14.png` exists in the export.

Reply:
That does not really matter. The replacement for existing attachments is done before this fix is applied. All available attachments are replaced but those with http:  or https: not.

Reply:
Okay, so if there is no `<ri:attachment>` node nested, the referenced file is not part of the export. Understood.